src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -221,6 +221,7 @@ const CCPTabPanel = ({ eventId }: { eventId: string }) => {
         Add CCP
       </Button>
       <OptionPopper
+        id={row || ''}
```

Comment:
this pattern is a little strange - i would prefer `id` to be a `number` type and there really isn't a reason for it to be converted into a `string` even when using it in `OptionPopper`; in any case this is fine for now - note that this may be refactored later on